docs/conf/prague/2020/welcome-wagon.rst

```diff
@@ -13,8 +13,7 @@ Write the Docs conference before.
 We are all first-timers at this year’s virtual conference. The Welcome
 Wagon is here for you, to help you make the most of it. We’d love to
 virtually meet you: feel free to reach out in Chat at the virtual
-conference if we can help
-make your time at the conference easier.
+conference if we can help make your time at the conference easier.
```

Comment:
During one of the planning calls, I think someone said that people weren't really reaching out to the Welcome Wagon specifically—sort of just throwing questions out there. It seems to me if nobody's going to email us, then we don't really need to provide an email

Reply:
That is definitely a self fulfilling prophecy though.

Reply:
I think we can just use the portland@writethedocs.org email address, and we can route WW-specific emails to the right people as needed.

Reply:
I've added prague@writethedocs.org as the email contact 👍 

I also think it makes sense to have it there, in case people can't get in touch using the conference software - for example if they're having trouble joining or something similar.